src/LIBSVM.jl

```diff
@@ -334,6 +309,7 @@ function svmtrain(
         weights = Float64[]
         reverse_labels = Bool[]
     else
+        check_train_input(X, y, kernel)
```

Comment:
I think `EpsilonSVR` and `NuSVR` also need this check?

Reply:
Good catch, I think so too. 

The check was originally in the `indices_and_weights` so it's been missing for `EpsilonSVR` and `NuSVR` for quite some time. Should I add it?

Reply:
yeah, add that check in this PR is fine for me.

Reply:
done